Run2 2017

rpatelCERN opened the pull request:
Sorry about that i made the PR to the wrong branch. I attach the same comments are as the previous PR:

Hey Kevin,
Here are a few changes that include:
Updating the scenarios.py for the _cff files for new 2017 collision datasets

Also I updated a few collection names based on the edm Collections in the latest data files.

I also updated the list of trigger names.
Let me know if you have any questions or suggestions

rpatelCERN committed:
Change in the Trigger Names

rpatelCERN committed:
Add Latest JSON

rpatelCERN committed:
Add 2017 Datasets

rpatelCERN committed:
Add 2017 Datasets

rpatelCERN committed:
Small Fixes: remove quotes around CutValues

rpatelCERN committed:
Update scenarios for 2017 data

rpatelCERN committed:
A version that runs

rpatelCERN committed:
A version that runs

rpatelCERN committed:
Version that Runs

rpatelCERN committed:
Working Version for Production

kpedro88 commented on Production/test/runMakeTreeFromMiniAOD_cfg.py:
```diff
@@ -48,7 +47,7 @@
 #temporary redirector fix
 #fastsim signal is phedexed to LPC Tier3
-redir=parameters.value("redir", "root://cmseos.fnal.gov/" if fastsim and signal else "root://cmsxrootd.fnal.gov/")
+redir=parameters.value("redir", "root://eoscms.cern.ch/" if fastsim and signal else "root://eoscms.cern.ch/")
```
nope, revert this
(the whole point of a command-line option is that you don't have to edit the file...)

rpatelCERN replied:
reverted

kpedro88 commented on TreeMaker/python/makeTreeFromMiniAOD_cff.py:
```diff
@@ -447,15 +448,13 @@ def makeTreeFromMiniAOD(
-        
         process.eeBadScFilter = filterDecisionProducer.clone(
-            trigTagArg1  = cms.string('TriggerResults'),
+            trigTagArg1  = cms.string("TriggerResults"),
```
Is there a reason this undoes the change in filterdecisionproducer_cfi.py? (and also for all the other MET filters...)

rpatelCERN replied:
reverted

kpedro88 commented on TreeMaker/python/makeTreeFromMiniAOD_cff.py:
```diff
@@ -204,6 +204,7 @@ def makeTreeFromMiniAOD(
     # default miniAOD tags
     JetTag = cms.InputTag('slimmedJets')
     JetAK8Tag = cms.InputTag('slimmedJetsAK8')
+    #JetAK8Tag = cms.InputTag('slimmedJetsAK8PFPuppiSoftDropPacked')
```
delete commented-out code

rpatelCERN replied:
removed

kpedro88 commented on TreeMaker/python/makeTreeFromMiniAOD_cff.py:
```diff
@@ -708,36 +707,39 @@ def makeTreeFromMiniAOD(
     
     # AK8 jet variables - separate instance of jet properties producer
     from TreeMaker.Utils.jetproperties_cfi import jetproperties
+    #JetAK8Tag = cms.InputTag('slimmedJetsAK8')
```
delete commented-out code

kpedro88 commented on Production/python/Run2017A-PromptReco-v3/HTMHT_cff.py:
```diff
@@ -0,0 +1,33 @@
+import FWCore.ParameterSet.Config as cms
```
2017A does not contain any certified data (and never will, afaik) - please remove

rpatelCERN replied:
Yes that is true, I removed that folder

kpedro88 commented on Production/python/scenarios.py:
```diff
@@ -60,6 +60,13 @@ def __init__(self,sname):
                           jecfile="data/jec/Summer16_23Sep2016AllV3_DATA",
                           residual=True,
                           era="Run2_25ns")
+        elif sname == "2017PromptCollisions":
```
* call this 2017BC (GT, etc. may change for later run periods)
* describe this scenario in README.md
* add an example to unitTest.py

kpedro88 commented on Production/python/scenarios.py:
```diff
@@ -60,6 +60,13 @@ def __init__(self,sname):
+            self.set_vars(globaltag="92X_dataRun2_Prompt_v8",
+                          tagname="RECO",
+                          jsonfile="data/Cert_294927-299042_13TeV_PromptReco_Collisions17_JSON.txt",
+                          jecfile="data/jec/Summer16_23Sep2016AllV3_DATA",
```
I doubt this is the right JEC for 2017 data... it actually doesn't look like there are 2017 JECs at this point, so probably best to leave this as `""` for now (which will avoid reapplying JECs at all, and just use the online)

rpatelCERN replied:
removed the JEC file. I agree we can go with the online jet energy instead

kpedro88 commented on Production/python/scenarios.py:
```diff
@@ -60,6 +60,13 @@ def __init__(self,sname):
+                          jsonfile="data/Cert_294927-299042_13TeV_PromptReco_Collisions17_JSON.txt",
+                          jecfile="data/jec/Summer16_23Sep2016AllV3_DATA",
+                          residual=True,
+                          era="Run2_25ns")
```
This should be `Run2_2017`

rpatelCERN replied:
I added the new JSON: https://cms-service-dqm.web.cern.ch/cms-service-dqm/CAF/certification/Collisions17/13TeV/PromptReco/Cert_294927-299649_13TeV_PromptReco_Collisions17_JSON.txt

kpedro88 commented on Production/python/Run2017B-PromptReco-v2/HTMHT_cff.py:
```diff
@@ -0,0 +1,44 @@
+import FWCore.ParameterSet.Config as cms
```
you need Run2017B-PromptReco-v1 also

rpatelCERN replied:
I was finding that the updated JSON does have runs in v2 of the 2017RunB dataset

kpedro88 replied:
There are certified runs in 2017B-v1 also, please add it

kpedro88 commented on Production/python/Run2017C-PromptReco-v2/MET_cff.py:
```diff
@@ -0,0 +1,69 @@
+import FWCore.ParameterSet.Config as cms
```
you need Run2017C-PromptReco-v1 also

rpatelCERN replied:
You are correct here, the current JSON will only have runs in v1.

kpedro88 commented on TreeMaker/python/makeTreeFromMiniAOD_cff.py:
```diff
@@ -708,36 +707,39 @@ def makeTreeFromMiniAOD(
+    process.JetsPropertiesAK8.NsubjettinessTau1 = cms.vstring('ak8PFJetsCHSValueMap:NjettinessAK8CHSTau1')
+    process.JetsPropertiesAK8.NsubjettinessTau2 = cms.vstring('ak8PFJetsCHSValueMap:NjettinessAK8CHSTau2')
+    process.JetsPropertiesAK8.NsubjettinessTau3 = cms.vstring('ak8PFJetsCHSValueMap:NjettinessAK8CHSTau3')
+    #process.JetsPropertiesAK8.bDiscriminatorSubjet1 = cms.vstring('SoftDrop','pfCombinedInclusiveSecondaryVertexV2BJetTags')
```
I think you can restore these by using `SoftDropPuppi`

rpatelCERN replied:
Hey Kevin, For the subjettiness and the AK8 jet mass we are currently using the pruned version of these variables (which is what was stored in the previous versions of Treemaker). These variables are still stored in the current datasets as user floats (which I got from your debug flag) : 

ak8PFJetsCHSValueMap:NjettinessAK8CHSTau1
 ak8PFJetsCHSValueMap:NjettinessAK8CHSTau2
ak8PFJetsCHSValueMap:NjettinessAK8CHSTau3
 ak8PFJetsCHSValueMap:ak8PFJetsCHSPrunedMass

I conferred with Andrew and the above variables are the ones we want to use.

kpedro88 commented on TreeMaker/python/makeTreeFromMiniAOD_cff.py:
```diff
@@ -708,36 +707,39 @@ def makeTreeFromMiniAOD(
-    process.JetsPropertiesAK8.bDiscriminatorSubjet2 = cms.vstring('SoftDrop','pfCombinedInclusiveSecondaryVertexV2BJetTags')
-    process.JetsPropertiesAK8.bDiscriminatorCSV = cms.vstring('pfBoostedDoubleSecondaryVertexAK8BJetTags')
+    #specify userfloat
+    #process.JetsPropertiesAK8.prunedMass = cms.vstring('prunedMass')
```
Just removed pruned mass and replace with `PuppiSoftDropMass` (userfloat `ak8PFJetsPuppiSoftDropMass`)

rpatelCERN replied:
Hey Kevin, For the subjettiness and the AK8 jet mass we are currently using the pruned version of these variables (which is what was stored in the previous versions of Treemaker). These variables are still stored in the current datasets as user floats (which I got from your debug flag) :

ak8PFJetsCHSValueMap:NjettinessAK8CHSTau1
ak8PFJetsCHSValueMap:NjettinessAK8CHSTau2
ak8PFJetsCHSValueMap:NjettinessAK8CHSTau3
ak8PFJetsCHSValueMap:ak8PFJetsCHSPrunedMass

I conferred with Andrew and the above variables are the ones we want to use.

kpedro88 commented on Utils/python/filterdecisionproducer_cfi.py:
```diff
@@ -1,8 +1,8 @@
 filterDecisionProducer = cms.EDProducer('FilterDecisionProducer',
-trigTagArg1  = cms.string('TriggerResults'),
-trigTagArg2  = cms.string(''),
+trigTagArg1  = cms.string('slimmedPatTrigger'),
```
this change breaks 2016 compatibility, so please remove all 2016 scenarios and input file lists (including from README, unit tests)

rpatelCERN replied:
This change is reverted, I checked and the "TriggerResults" collection works

kpedro88 commented on TreeMaker/python/makeTreeFromMiniAOD_cff.py:
```diff
@@ -708,36 +707,39 @@ def makeTreeFromMiniAOD(
+    #specify userfloat
+    #process.JetsPropertiesAK8.prunedMass = cms.vstring('prunedMass')
+    process.JetsPropertiesAK8.prunedMass = cms.vstring('ak8PFJetsCHSValueMap:ak8PFJetsCHSPrunedMass')
+    process.JetsPropertiesAK8.NsubjettinessTau1 = cms.vstring('ak8PFJetsCHSValueMap:NjettinessAK8CHSTau1')
```
I don't think these are right. It looks like the AK8 jet collection is now Puppi by default with PFCHS info stored as userfloats (reverse of the previous setup). So I think you want to use `NjettinessAK8Puppi:tau1`, etc.

(I turned on the debug flag in `JetProperties` and got this output:
```
subjetCollectionNames: SoftDropPuppi 
btagDiscriminatorNames: pfJetBProbabilityBJetTags pfJetProbabilityBJetTags pfTrackCountingHighEffBJetTags pfSimpleSecondaryVertexHighEffBJetTags pfSimpleInclusiveSecondaryVertexHighEffBJetTags pfCombinedSecondaryVertexV2BJetTags pfCombinedInclusiveSecondaryVertexV2BJetTags softPFMuonBJetTags softPFElectronBJetTags pfCombinedMVAV2BJetTags pfCombinedCvsLJetTags pfCombinedCvsBJetTags pfDeepCSVJetTags:probb pfDeepCSVJetTags:probc pfDeepCSVJetTags:probudsg pfDeepCSVJetTags:probbb pfBoostedDoubleSecondaryVertexAK8BJetTags pfBoostedDoubleSecondaryVertexAK8BJetTags 
userFloatNames: NjettinessAK8Puppi:tau1 NjettinessAK8Puppi:tau2 NjettinessAK8Puppi:tau3 ak8PFJetsCHSValueMap:NjettinessAK8CHSTau1 ak8PFJetsCHSValueMap:NjettinessAK8CHSTau2 ak8PFJetsCHSValueMap:NjettinessAK8CHSTau3 ak8PFJetsCHSValueMap:ak8PFJetsCHSPrunedMass ak8PFJetsCHSValueMap:ak8PFJetsCHSSoftDropMass ak8PFJetsCHSValueMap:eta ak8PFJetsCHSValueMap:mass ak8PFJetsCHSValueMap:phi ak8PFJetsCHSValueMap:pt ak8PFJetsPuppiSoftDropMass 
userIntNames: 
```
)

rpatelCERN replied:
Hey Kevin, For the subjettiness and the AK8 jet mass we are currently using the pruned version of these variables (which is what was stored in the previous versions of Treemaker). These variables are still stored in the current datasets as user floats (which I got from your debug flag) :

ak8PFJetsCHSValueMap:NjettinessAK8CHSTau1
ak8PFJetsCHSValueMap:NjettinessAK8CHSTau2
ak8PFJetsCHSValueMap:NjettinessAK8CHSTau3
ak8PFJetsCHSValueMap:ak8PFJetsCHSPrunedMass

I conferred with Andrew and the above variables are the ones we want to use.

kpedro88 replied:
These prompt 2017 ntuples won't be used for the boosted Higgs + MET analysis, will they? In the latest miniAOD, the default AK8 jets are now Puppi, so it seems like we should use the associated products from those jets. For 2016, we are still using PFCHS.

kpedro88 commented on Utils/src/TriggerProducer.cc:
```diff
@@ -141,10 +141,12 @@ TriggerProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup)
 
   //Find the matching triggers
   std::string testTriggerName;
+  //std::cout<<"Total Trigger Indices "<<parsedTrigNamesVec.size()<<std::endl;
```
remove commented-out code

kpedro88 commented on Utils/src/TriggerProducer.cc:
```diff
@@ -141,10 +141,12 @@ TriggerProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup)
     trigNamesVec->at(parsedIndex) = parsedTrigNamesVec[parsedIndex];
     for(unsigned int trigIndex = 0; trigIndex < trigNames.size(); trigIndex++){
       testTriggerName = trigNames.triggerName(trigIndex);
+    //std::cout<<"Trigger Name/ index "<<testTriggerName<<" "<<trigIndex<<std::endl;;
```
remove commented-out code

kpedro88 commented on TreeMaker/python/doZinvBkg.py:
```diff
@@ -63,7 +63,6 @@ def reclusterZinv(process, geninfo, residual, cleanedCandidates, suff, fastsim,
         )
     )
-    
     process.JetsPropertiesAK8Clean.prunedMass = cms.vstring('ak8PFJetsCHSCleanPrunedMass')
```
make the corresponding AK8 changes here (wrt my comments for the main collection, e.g. CHS -> Puppi, etc.)

rpatelCERN replied:
We are still sticking with pruned Mass for the boosted analyses, so for the control regions we still use the Pruned Mass (cleaned in the above for the photons)

kpedro88 commented on TreeMaker/python/triggerNameList.py:
```diff
@@ -1,49 +1,47 @@
-	'HLT_PFHT900_v',
-	'HLT_DoubleMu8_Mass8_PFHT300_v',
-	'HLT_DoubleEle8_CaloIdM_TrackIdM_Mass8_PFHT300_v',
+	'HLT_PFHT500_PFMET100_PFMHT100_IDTight_v',
```
Is this really the lowest-threshold HT-MHT trigger now? Wow...

rpatelCERN replied:
We also have MET/MHT only triggers which can be used for the signal trigger for lower MET regions :

        'HLT_PFMETNoMu110_PFMHTNoMu110_IDTight_v',
        'HLT_PFMETNoMu120_PFMHTNoMu120_IDTight_v',
        'HLT_PFMETNoMu130_PFMHTNoMu130_IDTight_v',
        'HLT_PFMETNoMu140_PFMHTNoMu140_IDTight_v',
        'HLT_PFMET110_PFMHT110_IDTight_v',
        'HLT_PFMET120_PFMHT120_IDTight_v',
        'HLT_PFMET130_PFMHT130_IDTight_v',
        'HLT_PFMET140_PFMHT140_IDTight_v',

rpatelCERN committed:
Latest Json File

rpatelCERN committed:
Include Revisions from Kevin

rpatelCERN committed:
Final Version for PR with Updated ReadME and unitTest

rpatelCERN committed:
Final Version for PR with Updated ReadME and unitTest

rpatelCERN committed:
Add Run2017C-PromptReco-v1

rpatelCERN commented:
Hey Kevin, 
Actually we were also asked to check the behaviour of our mass variable (which is the pruned mass) for the latest data. So for now we would like to keep the Subjettiness and the mass values as below. 

-Rishi
> On Aug 8, 2017, at 4:54 PM, Kevin Pedro <notifications@github.com> wrote:
> 
> @kpedro88 commented on this pull request.
> 
> In TreeMaker/python/makeTreeFromMiniAOD_cff.py <https://github.com/TreeMaker/TreeMaker/pull/236#discussion_r131936481>:
> 
> >              "bDiscriminatorCSV"    ,
>              "NumBhadrons"          ,
>              "NumChadrons"          ,
>          )
>      )
> -    #specify userfloats
> -    process.JetsPropertiesAK8.prunedMass = cms.vstring('ak8PFJetsCHSPrunedMass')
> -    process.JetsPropertiesAK8.NsubjettinessTau1 = cms.vstring('NjettinessAK8:tau1')
> -    process.JetsPropertiesAK8.NsubjettinessTau2 = cms.vstring('NjettinessAK8:tau2')
> -    process.JetsPropertiesAK8.NsubjettinessTau3 = cms.vstring('NjettinessAK8:tau3')
> -    process.JetsPropertiesAK8.bDiscriminatorSubjet1 = cms.vstring('SoftDrop','pfCombinedInclusiveSecondaryVertexV2BJetTags')
> -    process.JetsPropertiesAK8.bDiscriminatorSubjet2 = cms.vstring('SoftDrop','pfCombinedInclusiveSecondaryVertexV2BJetTags')
> -    process.JetsPropertiesAK8.bDiscriminatorCSV = cms.vstring('pfBoostedDoubleSecondaryVertexAK8BJetTags')
> +    #specify userfloat
> +    #process.JetsPropertiesAK8.prunedMass = cms.vstring('prunedMass')
> +    process.JetsPropertiesAK8.prunedMass = cms.vstring('ak8PFJetsCHSValueMap:ak8PFJetsCHSPrunedMass')
> +    process.JetsPropertiesAK8.NsubjettinessTau1 = cms.vstring('ak8PFJetsCHSValueMap:NjettinessAK8CHSTau1')
> These prompt 2017 ntuples won't be used for the boosted Higgs + MET analysis, will they? In the latest miniAOD, the default AK8 jets are now Puppi, so it seems like we should use the associated products from those jets. For 2016, we are still using PFCHS.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/TreeMaker/TreeMaker/pull/236#discussion_r131936481>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AEzr-0TUXNwiSSvu8OduWkQG47vFfOPoks5sWHazgaJpZM4OvidZ>.
>

kpedro88 commented:
okay, then we'll need to keep in mind to fix it to be congruent with the new default later

rpatelCERN committed:
Add Run2017B-v1

rpatelCERN commented:
Okay sure I added a folder:
Run2017B-23Jun2017-v1
to Production/python

You should now see the commit in the PR. 

-Rishi

> On Aug 8, 2017, at 4:58 PM, Kevin Pedro <notifications@github.com> wrote:
> 
> @kpedro88 commented on this pull request.
> 
> In Production/python/Run2017B-PromptReco-v2/HTMHT_cff.py <https://github.com/TreeMaker/TreeMaker/pull/236#discussion_r131937476>:
> 
> > @@ -0,0 +1,44 @@
> +import FWCore.ParameterSet.Config as cms
> There are certified runs in 2017B-v1 also, please add it
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/TreeMaker/TreeMaker/pull/236#discussion_r131937476>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AEzr-8Tz8y7aKY6Xr658guth7jzuhgDmks5sWHd8gaJpZM4OvidZ>.
>

kpedro88 commented:
@rpatelCERN I don't think that's the right one - we should probably just use Run2017B-PromptReco-v1, unless you explicitly know what the 23Jun2017 one is

rpatelCERN committed:
Add right version of Run2017B-PromptReco-v1

rpatelCERN commented:
Okay I think there might have been some Trigger updates between those datasets but I am not sure. I added Run2017B-PromptReco-v1 and removed the 23Jun2017.

> On Aug 8, 2017, at 5:23 PM, Kevin Pedro <notifications@github.com> wrote:
> 
> @rpatelCERN <https://github.com/rpatelcern> I don't think that's the right one - we should probably just use Run2017B-PromptReco-v1, unless you explicitly know what the 23Jun2017 one is
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/TreeMaker/TreeMaker/pull/236#issuecomment-320990375>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AEzr-zg7vgsuD6ffvb00xmFX0vIeFuLeks5sWH2BgaJpZM4OvidZ>.
>

kpedro88 commented:
you didn't remove the 23Jun2017 yet...

rpatelCERN committed:
Add right version of Run2017B-PromptReco-v1

rpatelCERN commented:
Fixed!
> On Aug 8, 2017, at 5:33 PM, Rishi Patel <rpatel@cern.ch> wrote:
> 
> Okay I think there might have been some Trigger updates between those datasets but I am not sure. I added Run2017B-PromptReco-v1 and removed the 23Jun2017.
> 
>> On Aug 8, 2017, at 5:23 PM, Kevin Pedro <notifications@github.com <mailto:notifications@github.com>> wrote:
>> 
>> @rpatelCERN <https://github.com/rpatelcern> I don't think that's the right one - we should probably just use Run2017B-PromptReco-v1, unless you explicitly know what the 23Jun2017 one is
>> 
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub <https://github.com/TreeMaker/TreeMaker/pull/236#issuecomment-320990375>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AEzr-zg7vgsuD6ffvb00xmFX0vIeFuLeks5sWH2BgaJpZM4OvidZ>.
>> 
>

kpedro88 commented on Utils/python/prescaleweightproducer_cfi.py:
```diff
@@ -6,6 +6,6 @@
    prescales = cms.InputTag("patTrigger"),
 
-   objects = cms.InputTag("selectedPatTrigger"),
+   objects = cms.InputTag("slimmedPatTrigger"),
```
this change also breaks 2016 compatibility, so please revert it if possible or remove all 2016 scenarios and input file lists (including from README, unit tests)

rpatelCERN committed:
For Incapitability

rpatelCERN commented:
That change is unfortunately necessary, so I commit a version that removes the 2016 datasets (MC/2016Data) and made the changes to the scenarios.py and the unitest.py

> On Aug 8, 2017, at 6:24 PM, Kevin Pedro <notifications@github.com> wrote:
> 
> @kpedro88 commented on this pull request.
> 
> In Utils/python/prescaleweightproducer_cfi.py <https://github.com/TreeMaker/TreeMaker/pull/236#discussion_r131962617>:
> 
> > @@ -6,6 +6,6 @@
>  
>     prescales = cms.InputTag("patTrigger"),
>  
> -   objects = cms.InputTag("selectedPatTrigger"),
> +   objects = cms.InputTag("slimmedPatTrigger"),
> this change also breaks 2016 compatibility, so please revert it if possible or remove all 2016 scenarios and input file lists (including from README, unit tests)
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/TreeMaker/TreeMaker/pull/236#pullrequestreview-54993488>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AEzr-xzW0PBP9Qju0zYe8nU3-FwpedWSks5sWIvCgaJpZM4OvidZ>.
>

kpedro88 commented:
also update readme

kpedro88 commented:
in principle, stuff in Production/test and Production/test/condorSub should also be updated

rpatelCERN committed:
Modify ReadME

rpatelCERN committed:
dictionary for Runs in Condor submission

rpatelCERN commented:
I added a commit to address this

> On Aug 8, 2017, at 9:13 PM, Kevin Pedro <notifications@github.com> wrote:
> 
> in principle, stuff in Production/test and Production/test/condorSub should also be updated
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/TreeMaker/TreeMaker/pull/236#issuecomment-321053683>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AEzr-ysB4TkDeEPhAJgfQShq124hRX6eks5sWLNVgaJpZM4OvidZ>.
>

rpatelCERN commented:
I added a commit to address this
> On Aug 8, 2017, at 9:10 PM, Kevin Pedro <notifications@github.com> wrote:
> 
> also update readme
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/TreeMaker/TreeMaker/pull/236#issuecomment-321052956>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AEzr-xvF6c2pP1EzcjlvqfqsfMmuSTZiks5sWLKtgaJpZM4OvidZ>.
>

kpedro88 commented on Production/test/condorSub/dict_2017B.py:
```diff
@@ -0,0 +1,11 @@
+flist = {
+    "scenario": "2017PromptCollisions",
+    "samples": [
+        ['Run2017B-PromptReco-v2.HTMHT'],
```
You should also be submitting 2017B-v1 (and eventually 2017C-v2, though none of it is certified yet)...

kpedro88 commented:
For the lumi calculation requested by @awhitbeck, you should also update https://github.com/TreeMaker/TreeMaker/blob/Run2_2017/Production/test/dataSamples.py

rpatelCERN commented:
Maybe Troy can do that, since he got the lumiCalc script to work?

-Rishi
> On Aug 9, 2017, at 4:41 PM, Kevin Pedro <notifications@github.com> wrote:
> 
> For the lumi calculation requested by @awhitbeck <https://github.com/awhitbeck>, you should also update https://github.com/TreeMaker/TreeMaker/blob/Run2_2017/Production/test/dataSamples.py <https://github.com/TreeMaker/TreeMaker/blob/Run2_2017/Production/test/dataSamples.py>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/TreeMaker/TreeMaker/pull/236#issuecomment-321276163>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AEzr-4NH8tSWJj4AnU07FPleiejU-fA1ks5sWcUEgaJpZM4OvidZ>.
>

rpatelCERN commented:
Sure I can do this.

-Troy

On 08/10/2017 06:16 AM, Rishi Patel wrote:
> Maybe Troy can do that, since he got the lumiCalc script to work?
>
> -Rishi
>> On Aug 9, 2017, at 4:41 PM, Kevin Pedro <notifications@github.com 
>> <mailto:notifications@github.com>> wrote:
>>
>> For the lumi calculation requested by @awhitbeck 
>> <https://github.com/awhitbeck>, you should also update 
>> https://github.com/TreeMaker/TreeMaker/blob/Run2_2017/Production/test/dataSamples.py
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub 
>> <https://github.com/TreeMaker/TreeMaker/pull/236#issuecomment-321276163>, 
>> or mute the thread 
>> <https://github.com/notifications/unsubscribe-auth/AEzr-4NH8tSWJj4AnU07FPleiejU-fA1ks5sWcUEgaJpZM4OvidZ>.
>>
>

kpedro88 commented:
going to merge this now, last few comments should be addressed in a followup PR

rpatelCERN commented:
Thanks! 
> On Aug 10, 2017, at 4:37 PM, Kevin Pedro <notifications@github.com> wrote:
> 
> going to merge this now, last few comments should be addressed in a followup PR
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/TreeMaker/TreeMaker/pull/236#issuecomment-321570752>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AEzr-8h_RwuQ12FKkE_y8fsMytM6EAV4ks5sWxWMgaJpZM4OvidZ>.
>
